### DISCUSSION TITLE
What is EP's current reliance on symbols?

### DISCUSSION THREAD

**Discussion by tallman69** [valinet/ExplorerPatcher#2279] [Q&A] [answered]
I know that work is being done to lessen EP's reliance on symbols. So, what is EP's current reliance on symbols?  The reason I am asking is because I updated one of my Windows 11 computers today to build 22621.2428, and EP seems to be working just fine even though the console reports `Failure in reading symbols "C: \Windows\system32\twinui.pcshell.dll"`.  Also I have not received the usual notification that symbols were successfully downloaded.  I am running the lates version of EP (22621.2361.58.4).

**Comment by SheepMoods** [comment]
Symbol files seems to be used only during the initial installation of the EP. But after updated EP may not work properly due to missing Symbols for some functions.

**Comment by Amrsatrio** [comment, marked as answer]
Probably this update is still really fresh out of the oven so it'll take a while for the symbols to be available. Anyways the start menu DLLs are the same as 22621.2361, so in the latest prerelease I've made EP to be able to process symbols for the start menu DLLs even though it couldn't for twinui.pcshell.dll.

Right now EP relies on symbols for working Win+X and Win10 alt tab on Canary builds, "open start menu to all apps" on all builds (I haven't made patterns for this), and "maximum number of frequent apps to show" or something (I haven't too).

I have a lot to say about Valinet's code tbh.............

**Comment by tallman69** [comment]
This morning I was able to download symbols for 22621.2428.

**Reply by Amrsatrio** [reply]
So what I said was correct. The symbols for this update were late, and that's enough to spawn couple issues in this repo 😂

**Reply by pyrates999** [reply]
Symbols still not available for windows 10 22h2 19045.3570.
